distrobox-enter

```diff
@@ -50,6 +50,8 @@ container_manager_additional_flags=""
 container_name=""
 container_name_default="my-distrobox"
 non_interactive=0
+non_interactive_assemble=0
```

Comment:
I'm not sure I like this
Enter should do enter things, and at most the "if not exist create a default"

I'd prefer the enter to stick to how it is now and not be entwined with assemble

Reply:
Okay that's understandable. Too bad. I will remove those changes.